src/creature_states_spdig.c

```diff
@@ -1746,9 +1796,11 @@ short creature_arms_trap(struct Thing *thing)
         return 0;
     }
     rearm_trap(traptng);
+    struct Dungeon* dungeon = get_dungeon(thing->owner);
     dungeon->lvstats.traps_armed++;
     creature_drop_dragged_object(thing, cratetng);
     delete_thing_structure(cratetng, 0);
+    thing_play_sample(traptng, 1000, NORMAL_PITCH, 0, 3, 0, 2, FULL_LOUDNESS);
```

Comment:
The same line is here and on line 1822. Is this on purpose or by accident?

Reply:
On purpose. One is first-person; the other isn't.